datapreview.tex

```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
+possible following the end of either ComCam on sky or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable the community to prepare to work with LSST data. 
+In the current schedule, which re-implements on-sky data-taking with ComCam for 2 months,  DP1 would be based on 2 months of ComCam data. 
+Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky or around System First Light.
 
-Note that the DP1 period of time during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken in the few days around System First Light.
-Table~\ref{tab:dp-one-products} presents a summary of the data products expected in DP1.
+In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
+At minimum,  DP1  will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Table~\ref{tab:data-preview-summary} presents a current best idea of  the data products that might comprise DP1. 
 
+A final decision on the dataset on which DP1 will be based will be taken in February 2024 following the Project's confirming decision about ComCam on-sky data-taking (\S~\ref{ssec:commissioning-schedule})
 
 \subsection{Data Preview 2}
 \label{sec:dp2}
 
-Data Preview 2 will serve a full consistent reprocessing of all data collected as part of the LSSTCam Science Validation Surveys (SV Surveys) together with any other science-grade commissioning data taken throughout the Science Optimization phase of commissioning, including the DP1 data. 
-Table~\ref{tab:dp-two-products} presents a summary of the data products expected in DP1.
+Data Preview 2 will serve a full consistent reprocessing of all data collected as part of the LSSTCam Science Validation Surveys (SV Surveys) together with any other science-grade 
+commissioning images taken throughout the Science Optimization phase of commissioning, including the DP1 data. 
+
+Table~\ref{tab:data-preview-summary} presents a summary of the data products expected in DP1.
 
 \subsection{Data Release 1}
 \label{sec:dr1}
 
 LSST Data Release 1 will be based on the first six months of data taken as part of the 10-year survey. 
-Data Release Processing of this dataset is estimated to to take six months, making the expected delivery date 1 year following the start of the 10 year survey. 
+Data Release Processing of this dataset is estimated to take six months, making the expected delivery date 1 year following the start of the 10 year survey. 
 DR1 will be the first Data Release in which all data products will be provided. 
 
 During routine LSST operations, prompt image data products will be made available 80 hours following camera readout.
```

Comment:
I think having this sentence appear to be part of the DR1 subsection is misleading.  This paragraph is about Prompt Products, not DR1.

Reply:
removed